Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -134,6 +134,20 @@ YUI.add('ez-binarybase-editview', function (Y) {
         },
 
         /**
+         * Check if the user is creating a new translation
+         *
+         * @method _isCreatingTranslation
+         * @protected
+         * @return {Boolean}
+         */
+        _isCreatingTranslation: function () {
+            var actualLanguageCode = this.get('languageCode'),
+                currentVersion = this.get('content').get('currentVersion');
+
+                return currentVersion.getTranslationsList().indexOf(actualLanguageCode) >= 0;
```

Comment:
this method does exactly the opposite of what its name suggests :) If the language code is in the list, then you are not creating a translation :-)

nitpick: indentation level